.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run format:fix
```

Comment:
I opted to remove the extra `echo` statements from the hook since `npm run format:fix` provides its own informational messages to the user.